app/views/partials/carouselitem.ejs

```diff
@@ -3,7 +3,7 @@
 	@mouseover="changeDocIndex(i)" @touchstart="changeDocIndex(i)">
 	<div class="carousel-item" v-for="(item, j) in doc.media" 
 	:style="{opacity: (sliderIndex === j ? 1 : 0 ) }">
-		<img class="d-block w-100" :src="item.image" :alt="'Slide '+ j"/>
+		<img class="d-block w-100" :src="item.image+'?version='+Math.random()" :alt="'Slide '+ j"/>
```

Comment:
The query, `?version=`, ensures updated images are always fetched from the server.